src/Command/PluginCommand.php

```diff
@@ -361,6 +361,8 @@ public function findPath(array $pathOptions, ConsoleIo $io): void
      */
     public function buildOptionParser(ConsoleOptionParser $parser): ConsoleOptionParser
     {
+        $parser = $this->_setCommonOptions($parser);
```

Comment:
Not all these options are relevant though. For example the `plugin` and `connection` options don't make sense when generating a plugin. Could we add only the theme option?

Reply:
That works for me. Can I use removeOption() to take the two out we don't need? Otherwise, I'm cutting and pasting to build up $bakeThemes again, or I lose the choices feature.

Reply:
I'll try refactoring CommonOptionsTrait a bit so I can call the $bakeThemes build from the PluginCommand. That'll keep it DRY and not require adding, then removing.

Reply:
I've pushed the refactored code which still ends up with some cut and paste from CommonOptionsTrait. Not sure that's better than the alternative of calling setCommonOptions, then removing the two that don't apply.

Let me know your opinion, I don't mind doing it again.

Reply:
Removing options works for me.